neuralpp/symbolic/expression.py

```diff
@@ -95,14 +101,19 @@ def _convert(cls, from_expression: Expression) -> Expression:
             return from_expression
         match from_expression:
             case Constant(value=value, type=type_):
-                return cls.new_constant(value, type_)
+                return cls.new_constant(from_expression.pythonize_value(value), type_)
```

Comment:
Aren't values of Constants always Python objects? Why do we need to pythonize it?
PS: later in the code I see that SymPy constants may have SymPy objects as values, but I think that is incorrect. The property `value` should provide a Python object for all `Expression` classes. `SymPy.value` should pythonize its SymPy object before returning it.

Reply:
fixed